packages/portal-proto/src/features/GenomicTables/GenesTable/utils.tsx

```diff
@@ -56,11 +56,12 @@ export const useGenerateGenesTableColumns = ({
             classNames={{
               input: "checked:bg-accent checked:border-accent",
             }}
+            aria-label={`checkbox for selecting genes table rows currently displayed`}
+            aria-checked={table.getIsAllRowsSelected()}
```

Comment:
this is not needed its handled automatically by the Checkbox

Reply:
should we just conditionally change the aria-label based off checked status?